### ISSUE TITLE
Update API to match new x-common file locations.

### ISSUE THREAD

**Issue by Insti** [exercism/x-api#132]
https://github.com/exercism/x-common/issues/269 proposes re-organising the way the files are stored in the x-common repository.

The API needs to be updated to look for files in the new locations and fall back to using the old 
locations if they are not found.

Hints:

The x-common repository is included as a git submodule called `metadata` in the x-api repository.

Files that reference the metadata directory are:
```
lib/xapi/problem.rb
lib/xapi/problems.rb
```
The changes will probably need to happen there.

**Comment by ramyaravindranath** [comment]
@kytrinyx  I am looking into this.

**Comment by Insti** [comment]
There is a current PR about moving the files: https://github.com/exercism/x-common/pull/351

To test the API with these changes applied you can do this from the x-api directory:
```
cd metadata
git checkout -b ErikSchierboom-new-exercises-format master
git pull https://github.com/ErikSchierboom/x-common.git new-exercises-format
```

**Comment by kytrinyx** [comment]
Fixed in #133
